tasks/configure.yml

```diff
@@ -8,11 +8,14 @@
 - name: create vertica dir
   file: path={{monasca_jar_dir}}/vertica state=directory owner=root group=root mode=755
   when: database_type == 'vertica'
-  register: create_directory
+
+- name: check if vertica jar exists
+  stat: path={{vertica_jdbc_location}}
+  register: status
 
 - name: place vertica jdbc
   command: cp {{vertica_jdbc_location}} {{monasca_jar_dir}}/vertica/vertica_jdbc.jar
-  when: create_directory|changed and database_type == 'vertica'
+  when: not status.stat.exists and database_type == 'vertica'
```

Comment:
Though it wasn't done this way before I wonder if the database_type check should be first. I'm not sure what will happen with status.stat if status doesn't exist. Other than that it looks good.